src/Common/Command/CcdaImport.php

```diff
@@ -44,8 +44,12 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $output->writeln('document_id parameter is missing (required), so exiting');
             return 2;
         }
+        if (empty($input->getOption('auth_name'))) {
+            $output->writeln('auth_name parameter is missing (required), so exiting');
+            return 2;
+        }
+        $_SESSION['authUser'] = $input->getOption('auth_name');
 
-        $GLOBALS['modules_application']->getServiceManager()->build(CarecoordinationTable::class)->import($input->getOption('document_id'));
```

Comment:
@bradymiller Couldn't understand why this was used twice. Hope I didn't miss why and guessing was just overooked...